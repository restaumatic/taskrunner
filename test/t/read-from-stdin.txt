# Regression test: in early versions we opened `/dev/null` for stdin in write mode (should be read).
# Discovered via Spring (of Ruby on Rails) throwing and exception and hanging.
# With bug present, outputs additional error message:
# [toplevel] stderr | bash: line 1: read: read error: 0: Bad file descriptor
# Without bug, the read still fails (EOF), but there's no extra error message.
read foo || echo "Nothing there"
