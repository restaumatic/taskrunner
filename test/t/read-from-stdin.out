-- output:
[toplevel] stdout | Nothing there
